interface/src/app/node/node.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.node', [
+  'ui.router',
+  'placeholders',
```

Comment:
Good to know. Thanks. (I had to confess that I never really understood what **angular-placeholders** module  exactly does.)